docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS
+description: Learn where and what to install for .NET Core on Windows, Linux, and macOS versions. Discover the dependencies required to develop, deploy, and run .NET Core apps.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install .NET Core
+
+This article provides information on downloading .NET Core and what is required to install .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and apps.
+
+.NET Core is made up of two components, the runtime and the SDK. The **runtime** is required to run a .NET Core app, when that app doesn't include the runtime. The **SDK** is required to build .NET Core apps and libraries.
+
+There are different ways to develop .NET Core apps:
```

Comment:
Does Docker fit it in here at all?  Docker is useful and often used while developing .NET Core apps.  VS Code in particular makes developing with Docker images easy and attractive.  While you don't technically install images, it is a way to acquire the SDK for development purposes.